src/database/services/PanelSetService.ts

```diff
@@ -0,0 +1,47 @@
+import { Sequelize } from "sequelize";
+import { PanelSet } from "../models/panelSet.model";
+import { User } from "../models/user.model";
+
+interface PanelSetConfig {
+    username: string
+}
+
+class PanelSetService {
+    /**
+     * Create a new panel set
+     * @param {} panelSet 
+     */
+    static async createPanelSet(panelSet: PanelSetConfig) {
```

Comment:
Just wondering if there is anywhere the id of the created panelset is getting returned for saving somewhere, as without it the `getPanelSetByID` would not be relevant.